device/rsvp/rsvpEgressLsp.yaml

```diff
@@ -0,0 +1,49 @@
+components:
+  schemas:
+    Rsvp.LspInterface.EgressP2PLSP:
+      description: >-
+        Configuration for RSVP Egress P2P LSPs.
+      type: object
+      properties:
+        refresh_interval:
+          description: >-
+            The time in seconds between transmissions of RESV Refreshes.
```

Comment:
The time in seconds between successive transmissions of RESV Refreshes.

Reply:
Done.